util/devel/chplspell-dictionary

```diff
@@ -7594,6 +7594,8 @@ iolittle
 iomanip
 iomode
 iomodes
+ioMode
+ioModes
```

Comment:
Similar to the `openMemFile` change, these updates shouldn't be necessary -- for the same reasons, and also because (as I didn't mention) the comparisons with the dictionary (after that tokenization) are done case-insensitively, so these new entries are equivalent to the preceding two entries.